cmake/AddGeneratedCodeLib.cmake

```diff
@@ -95,6 +72,10 @@ file(STRINGS ${CODER_DEFINES_FILE} CODER_DEFINES)
 # Model includes
 set(CODER_INCLUDES "${AUTOGEN_ABSPATH}")
 
+# TODO: figure out what is the best way to get Matlab_ROOT_DIR.
+#       Should we find it here, use an input variable, of inheriting it
+#       from the scope?
```

Comment:
I would say "inheriting it from the scope, but checking the `MATLAB_FOUND` variable or similar, and if it is not defined print a clear error sayng that one should call `find_package(Matlab ..)` before using this CMake macro.

Reply:
Added the check, thanks for the advice.